english/expelliarmus/expelliarmus.md

```diff
@@ -0,0 +1,4 @@
+Use this disarming charm, also known as the Expelliarmus spell
+as a defensive tactic to force the marshmallow to release you.
+This has worked for Harry Pothead but may not for you.
+Given your current level of intoxication. In that case you may just need to take the [red pill](red-pill/red-pill.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.